swift/Workflow/Tooling/Templates/Workflow (Verbose).xctemplate/___FILEBASENAME___Workflow.swift

```diff
@@ -0,0 +1,94 @@
+//  ___FILEHEADER___
+
+import Workflow
+import WorkflowUI
+import ReactiveSwift
+import Result
+
+
+// MARK: Input
+
+struct ___VARIABLE_productName___Workflow: Workflow {
+
+}
+
+
+// MARK: Output
+
+extension ___VARIABLE_productName___Workflow {
```

Comment:
Now that the protocol is shorter I would bring all of this together (not putting everything in a separate extension).

Reply:
For all the ones of these I've build, I've kept the separation. Especially since as the complexity grows of a Workflow, the separation I find helps readability